department-of-veterans-affairs/caseflow
# Shalliburton/13208 fix disposition undefined

Resolves #13208 

### Description
Added error handling to prevent modal from crashing and catch error when hearing or hearing task association is missing

### Acceptance Criteria
- [ ] User can change hearing disposition

### Testing Plan
1. Select judge "BVATWARNER": http://localhost:3000/test/users
1. Navigate to queue:  http://localhost:3000/queue
1. Change view to the admin teams view
1. Select one of the cases with the task set to "Change Hearing Disposition"
1. Change the hearing disposition and ensure you do not receive an error
